app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,29 @@
 <!DOCTYPE html>
 <html>
```

Comment:
require formatting just.

Reply:
it doesn't have formatting issues in sublime :P, idk why it shows issues here